cirq-core/cirq/contrib/svg/svg.py

```diff
@@ -23,6 +23,10 @@ def fixup_text(text: str):
     if '[cirq.VirtualTag()]' in text:
         # https://github.com/quantumlib/Cirq/issues/2905
         return text.replace('[cirq.VirtualTag()]', '')
+    if '<' in text:
+        return text.replace('<', '&lt;')
+    if '>' in text:
+        return text.replace('>', '&gt;')
```

Comment:
Looks like this replaces only one of the `<>` characters, we can still get some grief for `wire_symbol = '>-<'`

How about
```patch
diff --git a/cirq-core/cirq/contrib/svg/svg.py b/cirq-core/cirq/contrib/svg/svg.py
index b08c8a71..17e8b4ec 100644
--- a/cirq-core/cirq/contrib/svg/svg.py
+++ b/cirq-core/cirq/contrib/svg/svg.py
@@ -23,10 +23,7 @@ def fixup_text(text: str):
     if '[cirq.VirtualTag()]' in text:
         # https://github.com/quantumlib/Cirq/issues/2905
         return text.replace('[cirq.VirtualTag()]', '')
-    if '<' in text:
-        return text.replace('<', '&lt;')
-    if '>' in text:
-        return text.replace('>', '&gt;')
+    text = text.replace('<', '&lt;').replace('>', '&gt;')
     return text
```

Reply:
done in #6056